app/controllers/pages_controller.rb

```diff
@@ -22,6 +22,12 @@ def about_listings
     set_surrogate_key_header "about_listings_page"
   end
 
+  def badge
+    @html_variant = HtmlVariant.find_for_test([], "badge_landing_page")
+    render layout: false
+    set_surrogate_key_header "badge_page"
+  end
```

Comment:
While working on this feature, I noticed that this file's methods were _mostly_ alphabetized, but not entirely. That said, I went ahead and rearranged some of the methods (`#badge`, `#privacy`, and `#terms`) to better follow the alphabetization within the file.